dev/io.openliberty.microprofile.jwt.2.1.internal_fat_tck/publish/servers/jwt20tckAudEnv/jvm.options

```diff
@@ -0,0 +1 @@
+-Dcom.ibm.tools.attach.enable=yes
```

Comment:
Please rename the folders where these server files are from jwt20* to jwt21*. For example, from jwt20tckAudEnv to jwt21tckAudEnv. Same for all subsequent servers. Thanks!